config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -65,3 +65,31 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-image-builder
       testgrid-tab-name: pr-json-sort-check
+  - name: packer-validate
+    decorate: true
+    run_if_changed: 'images/capi/Makefile|images/capi/packer/.*'
+    decoration_config:
```

Comment:
we should set this to 

```
always_run: false
optional: true
```

so it can be merged and used to validate https://github.com/kubernetes-sigs/image-builder/pull/546, and then once https://github.com/kubernetes-sigs/image-builder/pull/546 merges we can change it to required and run_if_changed

Reply:
@CecileRobertMichon Thanks for keeping me honest. PTAL?